lib/handbook/addon/docs/testing/template.md

```diff
@@ -1,3 +1,32 @@
 # Testing
 
-TODO
+## Test categories
+
+### By what they try to determine
+We organize the tests in the hierarchy by this set of categories.
+
+#### Acceptance test
+These are a type of Application test (see below) that looks at the behavior of the whole app from a user's perspective. Ideally, when adding a new page, the Acceptance tests should be the first thing you write after you get all of the appropraite information from Product and preferably after consulting with QA. The handy thing about acceptance tests is that it's behavior driven, so it shouldn't matter what the underlying implementation of the models or the components are. If the OSF API is defined, you should be able to define what the test selectors are and how they react to various situations, then fill in the functionality of everything else to make those tests pass.
```

Comment:
> If the OSF API is defined, you should be able to define what the test selectors are and how they react to various situations, then fill in the functionality of everything else to make those tests pass.

I think it would be clearer to move this to a new paragraph and expand on it slightly. If we hadn't already talked about it, I might have trouble understanding what you mean.